uk.co.compucorp.civicrm.hrleaveandabsences/xml/schema/CRM/HRLeaveAndAbsences/AbsencePeriod.entityType.php

```diff
@@ -0,0 +1,11 @@
+<?php
+// This file declares a new entity type. For more details, see "hook_civicrm_entityTypes" at:
+// http://wiki.civicrm.org/confluence/display/CRMDOC/Hook+Reference
+return array (
+  0 => 
+  array (
+    'name' => 'AbsencePeriod',
+    'class' => 'CRM_HRLeaveAndAbsences_DAO_AbsencePeriod',
+    'table' => 'civicrm_absenceperiod',
+  ),
+);
```

Comment:
This file was automatically generated wi GenCode.php and it will be overwritten whenever I generate or update any of the xml schema files.